src/app/youtube/pipes/sort-views.pipe.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { SortViewsPipe } from './sort-views.pipe';
+
+describe('SortViewsPipe', () => {
+  it('create an instance', () => {
```

Comment:
this test makes nothing important. please remove it or test `transform ` method. 
[https://angular.io/guide/testing-pipes](https://angular.io/guide/testing-pipes)

Reply:
I removed it